### PR TITLE
Gemstones: add iftop entry

### PR DESCRIPTION
**Pull request by neelchauhan**
#### Author checklist (Completed by original Author)
- [X] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [X] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/gemstones/iftop.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+|-u UNIT |Display traffic rates with the given unit|
+|-B UNIT |Synonym for -u|
+|-i INTERFACE |Interface to measure|
+|-f FILTER CODE |Use the following filter code|
+|-F NET/MASK |Only measure traffic going to the specified IPv4 network|
+|-G NET/MASK |Only measure traffic going to the specified IPv6 network|
+|-c config |Use the following configuration file|
+|-t |Use non-ncurses mode|
+
+The units for the **-M** flag is as follows:
```
Comment (SergeCroise) [inline docs/gemstones/iftop.md:39]:
```suggestion
The units for the **-M** flag are as follows:
```

File: docs/gemstones/iftop.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+|Unit|Meaning|
+|---|---|
+|K |Kilo|
+|M |Mega|
+|G |Giga|
+
+The units for the **-u** flag is as follows:
```
Comment (SergeCroise) [inline docs/gemstones/iftop.md:47]:
```suggestion
The units for the **-u** flag are as follows:
```

File: docs/gemstones/iftop.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+---
+title : nload - Live Per-Connection Bandwidth Statistics
```
Comment (SergeCroise) [inline docs/gemstones/iftop.md:2]:
```suggestion
title: nload - Live Per-Connection Bandwidth Statistics
```

File: docs/gemstones/iftop.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+author : Neel Chauhan
```
Comment (SergeCroise) [inline docs/gemstones/iftop.md:3]:
```suggestion
author: Neel Chauhan
```

File: docs/gemstones/iftop.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+contributors:
+date : 2024-02-24
```
Comment (SergeCroise) [inline docs/gemstones/iftop.md:5]:
```suggestion
date: 2024-02-24
```

**Comment by neelchauhan** [comment]
Hi all, I have committed the suggestions made.

File: docs/gemstones/iftop.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+---
+
```
[the remark this answers is not in the document]
Comment (neelchauhan) [reply docs/gemstones/iftop.md:7]: Yes. Changed it.

**Pull request merged**